lib/args.js

```diff
@@ -69,6 +70,10 @@ function args(argv) {
     opts['web-security'] = opts['web-security'] === 'true';
   }
 
+  if (opts.hasOwnProperty('ignore-ssl-errors')) {
+    opts['ignore-ssl-errors'] = opts['ignore-ssl-errors'] === 'true';
```

Comment:
This seems unnecessary. When you invoke `mochify --ignore-ssl-errors`, then `opts['ignore-ssl-errors']` is already set to the boolean `true`. With this one is required to write `mochify --ignore-ssl-errors true` to make it work. See `--node` and `--wd`.